index.html

```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leading Eagle Brand</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="https://use.typekit.net/xxz1eii.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+</head>
+
+<body>
+    <header>
+
+        <nav class="navbar">
+            <div class="top-nav">
+                <div class="top-nav_container d-flex">
+                    <ul class="top-nav_content d-flex">
+                        <li class="top-nav-links"><i class="fa-brands fa-facebook-f topicon"></i></li>
+                        <li class="top-nav-links"><i class="fa-brands fa-twitter topicon"></i></li>
+                        <li class="top-nav-links"><span class="toplink">English</span></li>
+                        <li class="top-nav-links"><span class="toplink">My Page</span></li>
+                        <li class="top-nav-links"><span class="toplink">Logout</span></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="navbar-innercontainer ">
+                <a class="brand" href="/index.html"><img src="/Assets/Logos/navlogo.png" class="nav-logo"
+                        alt="nav-logo"></a>
+                <ul class="nav-items" id="nav-menu">
+                    <li class="nav-link" id="navlink"><a href="/index.html" class="link active">Home</a></li>
+                    <li class="nav-link" id="navlink">
+                        <a href="AboutPage.html" class="link">About</a>
+                    </li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">Join</a></li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">Sponsor</a></li>
+                    <li class="nav-link" id="navlink"><a href="#contact" class="link">News</a></li>
+                    <li class="nav-link" id="navlink">
+                        <div class="campaign_container">
+                            <span>CC Campaign</span>
+                        </div>
+                    </li>
+                </ul>
+                <div class="baricon" id="nav-toggler">
+                    <div class="bar1"></div>
+                    <div class="bar2"></div>
+                    <div class="bar3"></div>
+                </div>
+            </div>
+        </nav>
+    </header>
```

Comment:
- [ ] Nice work adding the social media bar as requested by the previous reviewer :clap: However It would be nice to align the social media link with your navigation bar link to follow the provided design.

### Provided design :point_down: 
![Screenshot from 2023-04-22 16-07-05](https://user-images.githubusercontent.com/91142432/235314363-4f223193-5b40-4406-bc6d-12c3d84ca6ea.png)

### Your design :point_down: 
![Screenshot from 2023-04-29 16-52-48](https://user-images.githubusercontent.com/91142432/235314399-c38f1195-1c3e-43ea-8c16-afc3707759af.png)